features/conversation-list/hooks/use-conversation-is-pinned.ts

```diff
@@ -0,0 +1,25 @@
+import { useCurrentAccount } from "@/data/store/accountsStore";
+import { getConversationDataQueryOptions } from "@/queries/use-conversation-data-query";
+import { useQuery } from "@tanstack/react-query";
+import { ConversationTopic } from "@xmtp/react-native-sdk";
+
+export function useConversationIsPinned(args: {
+  conversationTopic: ConversationTopic;
+}) {
+  const { conversationTopic } = args;
+
+  const currentAccount = useCurrentAccount();
+
+  const { data: isPinned } = useQuery({
+    ...getConversationDataQueryOptions({
```

Comment:
tiny nicer syntax here could be asking for select in `getQueryOptions` and defaulting to returning data: any so that we could just say `useQuery(getQueryOptions({selector: data => data.property, account, topic, context}))` rather than needing to destructure AND create the args